### ISSUE TITLE
'npm test' fails on master branch with node v7.0

### ISSUE THREAD

**Issue by rashmihunt** [strongloop/loopback-datasource-juggler#1370]
- Install node v7.0
- git clone https://github.com/strongloop/loopback-datasource-juggler.git
- npm i
- npm test

```  1822 passing (6s)
  133 pending
  1 failing

  1) ModelBuilder processing json files allows model extension:

      AssertionError: 6 == 7
      + expected - actual

      -6
      +7

      at Context.<anonymous> (test/loopback-dl.test.js:1841:12)```

**Comment by dhmlau** [comment]
https://github.com/strongloop-internal/scrum-apex/issues/209

**Comment by dhmlau** [comment]
Fixed in strongloop-internal/scrum-apex#209.  Closing this issue.
